src/parsing.rs

```diff
@@ -1288,7 +1283,8 @@ fn group_entry_to_type(
     parent_visitor: &ParentVisitor,
     entry: &GroupEntry,
 ) -> RustType {
-    let ret = match entry {
+    //println!("group_entry_to_typename({:?}) = {:?}\n", entry, ret);
```

Comment:
```suggestion
```
This was only a temporary anyway so it could be printed so we might as well remove this scar tissue comment too if we're removing that temporary.